src/atoms/Space/__tests__/Space.web.test.js

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import { render } from '@testing-library/react';
 import View from '../../View';
 import Space from '../index';
+import { renderWithTheme } from '../../../_helpers/testing';
 
 beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
 afterAll(() => jest.restoreAllMocks());
 
 describe('Renders <Space /> correctly', () => {
```

Comment:
modify the stories as per the convention discussed

Reply:
Done

Reply:
I can't see them as per convention. Please refer the slack thread again? It has proper details with screenshots and text